BD-3658 Create tutorial for Liquid conditional logic (#8862)
braze-inc/braze-docs: _docs/_user_guide/personalization_and_dynamic_content/liquid/conditional_logic.md

{% endif %}
```

### Conditional tags

> #### `if` and `elseif`

Reviewer: ```suggestion
#### `if` and `elsif`
```

> Conditional logic begins with the `if` tag, which states the first condition to check. Subsequent conditions use the `elseif` tag and will be checked if the previous conditions are not met. In this example, if a user's device is not set to English this code will check to see if the user's device is set to Spanish or Chinese. If the user's device meets one of these conditions, the user will receive a message in the relevant language.

Reviewer: `this code will check to see if the user's device is set to Spanish or Chinese.`
This could be a bit too strict but this code followed this statement it would be `elsif ${language} == 'es' or elsif ${language} == 'zh'`. It would be more accurate to say it checks to see if the device is set to Spanish, and if that fails then it checks Chinese.

> Conditional logic begins with the `if` tag, which states the first condition to check. Subsequent conditions use the `elseif` tag and will be checked if the previous conditions are not met. In this example, if a user's device is not set to English this code will check to see if the user's device is set to Spanish or Chinese. If the user's device meets one of these conditions, the user will receive a message in the relevant language.

Reviewer 2: ```suggestion
Conditional logic begins with the `if` tag, which states the first condition to check. Subsequent conditions use the `elsif` tag and will be checked if the previous conditions are not met. In this example, if a user's device is not set to English this code will check to see if the user's device is set to Spanish or Chinese. If the user's device meets one of these conditions, the user will receive a message in the relevant language.
```

> Conditional logic begins with the `if` tag, which states the first condition to check. Subsequent conditions use the `elseif` tag and will be checked if the previous conditions are not met. In this example, if a user's device is not set to English this code will check to see if the user's device is set to Spanish or Chinese. If the user's device meets one of these conditions, the user will receive a message in the relevant language.

Author: ```suggestion
Conditional logic begins with the `if` tag, which states the first condition to check. Subsequent conditions use the `elseif` tag and will be checked if the previous conditions are not met. In this example, if a user's device is not set to English this code will check to see if the user's device is set to Spanish, and if that fails, then it will check if the device is set to Chinese. If the user's device meets one of these conditions, the user will receive a message in the relevant language.
```

> Conditional logic begins with the `if` tag, which states the first condition to check. Subsequent conditions use the `elsif` tag and will be checked if the previous conditions are not met. In this example, if a user's device isn't set to English this code will check to see if the user's device is set to Spanish, and if that fails, will check if the device is set to. If the user's device meets one of these conditions, the user will receive a message in the relevant language.

Reviewer 3: ```suggestion
Conditional logic begins with the `if` tag, which states the first condition to check. Subsequent conditions use the `elsif` tag and will be checked if the previous conditions are not met. In this example, if a user's device isn't set to English, this code will check to see if the user's device is set to Spanish, and if that fails, it will check if the device is set to. If the user's device meets one of these conditions, the user will receive a message in the relevant language.
```

#### `else`

> You have the option to include an `{% else %}` statement in your conditional logic. If none of the conditions that you set are met, the `{% else %}`  statement specifies the message that should send. In this example, we default to English if a user’s language is not English, Spanish, or Chinese.

Reviewer: ```suggestion
You have the option to include an `{% else %}` statement in your conditional logic. If none of the conditions that you set are met, the `{% else %}` statement specifies the message that should send. In this example, we default to English if a user’s language is not English, Spanish, or Chinese.
```

> You have the option to include an `{% else %}` statement in your conditional logic. If none of the conditions that you set are met, the `{% else %}` statement specifies the message that should send. In this example, we default to English if a user’s language is not English, Spanish, or Chinese.

Reviewer 3: ```suggestion
You have the option to include an `{% else %}` statement in your conditional logic. If none of the conditions that you set are met, the `{% else %}` statement specifies the message that should be sent. In this example, we default to English if a user’s language is not English, Spanish, or Chinese.
```

#### `endif`

The `{% endif %}` tag signals that you've finished your conditional logic. You must include the `{% endif %}` tag in any message with conditional logic. If you don't include an `{% endif %}` tag in your conditional logic, you'll get an error as Braze will be unable to parse your message.

> ### Tutorial: Deliver location-based content

Reviewer: Is location a default user attribute? https://www.braze.com/docs/user_guide/personalization_and_dynamic_content/liquid/supported_personalization_tags/#summary-of-supported-tags Don't see it here so if someone follows this exactly it wouldn't work.

Reviewer 2: Yes, location isn't a default attribute in Braze. But if the reader understands they can initialize a variable `location`, then it won't be an issue.

Reviewer 2: Oh, yes, since `location` isn't a Braze default liquid variable, the example with `${location}` won't work. Two options here
* Pick a different Braze default liquid variable, for example, `${city}`
* We can initialize a variable e.g. `{% assign location="..." %}`, then use the location in liquid like `{{location}}`. 

For this tutorial, for simplicity, I will suggest we pick a different liquid variable

> When you're finished with this tutorial, you'll be able to use tags with "if", "elsif" and "else" statements to deliver content based on a user's location.

Reviewer 3: ```suggestion
When you're finished with this tutorial, you'll be able to use tags with "if", "elsif", and "else" statements to deliver content based on a user's location.
```

> 1. Begin with an `if` tag to establish what message should send when the user's city is in New York. If the user's city is New York, this first condition is met and the user will receive a message specifying their New Yorker identity.

Reviewer 3: ```suggestion
1. Begin with an `if` tag to establish what message should be sent when the user's city is in New York. If the user's city is New York, this first condition is met and the user will receive a message specifying their New Yorker identity.
```

```liquid

> {% if {{${location}}} == "New York" %}

Reviewer 2: The extra curly braces are not necessary. 
```suggestion
{% if ${location} == "New York" %}
```

  🎉 Hey there, New Yorker! We're excited to offer you a special deal! 
  Get 20% off your next sandwich at your local Sandwich Emperor. 
  Just show this message at the counter to redeem your offer!
```

{: start="2"}

> 2. Next, use the `elseif` tag to establish what message should send if the user's city is in Los Angeles.

Reviewer 3: ```suggestion
2. Next, use the `elseif` tag to establish what message should be sent if the user's city is in Los Angeles.
```

```liquid
{% elsif ${city} == "Los Angeles" %}
  🌞 Hello, Los Angeles! Enjoy a sunny day with a delicious sandwich! 
  Present this message at our LA restaurant for a 20% discount on your next order!
```

{: start="3"}

> 3. Let's use another `elseif` tag to establish what message should send if the user's city is in Chicago.

Reviewer 3: ```suggestion
3. Let's use another `elseif` tag to establish what message should be sent if the user's city is in Chicago.
```

```liquid

> {% elsif {{${location}}} == "Chicago" %}

Reviewer 2: ```suggestion
{% elsif ${location} == "Chicago" %}
```

  🍕 Chicago, we have a treat for you! 
  Swing by our restaurant and get 20% off your favorite sandwich. 
  Just show this message to our staff!
```

{: start="4"}

> 4. Now, let's use the `{% else %}` tag to specify what message should send if the user's city isn't in San Francisco, New York, or Chicago.

Reviewer 3: ```suggestion
4. Now, let's use the `{% else %}` tag to specify what message should be sent if the user's city isn't in San Francisco, New York, or Chicago.
```
